### ISSUE TITLE
Allow room visitors to cycle through pre-approved scenes

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2464]
**Is your feature request related to a problem? Please describe.**
Currently, room visitors can't change the scene in a room. It would be very helpful if people were able to take self-guided scene tours to pre-approved scenes. 

This would be helpful if you wanted people to be able to tour several types of environments without having an admin present, or for self-guided storytelling purposes. 

**Describe the solution you'd like**
It would be helpful if visitors to a Hubs room could change the scene to one that is built in using Spoke without having to be an admin. 

or 

Maybe there could be a little indicator that would let you move through a preapproved list of scenes. 

When a room is empty, the scene could reset to the first one.

**Additional context**
A community member requested this feature, and based on previous meetings with folks, I think this would be helpful for a range of use cases. 

This would also be super helpful for me so people could revisit tours of hubs rooms I've made for demo purposes.

**Comment by Anandavadivelan** [comment]
Yes I think that would be necessary feature. I already have two separate use cases where this is needed and not having that is limiting on Hubs.
